curiouslearning/FeedTheMonsterJS
# Feat: revamp feed the monster text design and animation (FM-272)

# Changes
- Update the title text effect in the start screen
- Created a separate div for the title text
- Created a reusable function to apply font so that feedback and start scene title share the same code

# How to test
- Make sure start screen title is working properly or displaying properly in all languages.

Ref: [FM-272](https://curiouslearning.atlassian.net/browse/FM-272)


[FM-272]: https://curiouslearning.atlassian.net/browse/FM-272?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
